### PR TITLE
Initial server setup

### PR DESCRIPTION
**Pull request by shrikantr-webonise**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+node_modules
+*~
+cov-*
+coverage
+plato
+.idea
+fhc-debug.log
```
Comment (eliza-abraham) [inline .gitignore]: dont need this i suppose

File: app/routes/index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+/*jshint expr: true */
+
+'use strict';
+
+let baseRoute = require('./base');
+
+exports.init = (router) => {
+  //log message in console on every request
+  router.use((req, res, next) => {
+    console.log('%s %s %s', req.method, req.url, req.path);
```
Comment (eliza-abraham) [inline app/routes/index.js]: do we want to add a logger instead of console.log?

Comment (vjnathe-webonise) [reply app/routes/index.js]: winston logger

Comment (shrikantr-webonise) [reply app/routes/index.js]: Added Winston logger.

**Comment by shrikantr-webonise** [comment]
Closing this PR as this change is a part of another PR: https://github.com/shrikantr-webonise/creo-invent-node/pull/2

**Pull request closed without merging**